### PR TITLE
Add a note about running stack on other Linux distros

### PR DESCRIPTION
**Pull request by bd-dean**
This PR needs discussion before merging.

### CODE REVIEW

File: source/clear-linux/tutorials/dlrs/dlrs.rst [hunk 1]
```diff
@@ -95,11 +95,16 @@ Version compatibility
 
 We validated these steps against the following software package versions:
 
-* |CL| 26240 (Lower version not supported)
+* |CL| 26240 (Minimum supported version)
```
Comment (puneetse) [inline source/clear-linux/tutorials/dlrs/dlrs.rst]: Are these components all >= minimum requirements, or very specifically those versions only?
I understand the point of this is the *tested* version, but is the intent that newer version should also work?

Comment (bd-dean) [reply source/clear-linux/tutorials/dlrs/dlrs.rst]: Newer versions should work.

**Comment by bd-dean** [comment]
Added section with "uninstallation" instructions.

**Comment by mvincerx** [comment]
LGTM. Thanks, @bd-dean.

**Pull request merged**